cypress/e2e/tests/pages/manager/repositories.spec.ts

```diff
@@ -9,6 +9,30 @@ import { CLUSTER_REPOS_BASE_URL } from '@/cypress/support/utils/api-endpoints';
 const chartBranch = `release-v${ CURRENT_RANCHER_VERSION }`;
 const gitRepoUrl = 'https://github.com/rancher/charts';
 
+describe('Visual Testing', { testIsolation: 'off', tags: ['@manager', '@adminUser'] }, () => {
+  before(() => {
+    cy.login();
+  });
+  it('validating repositories page with percy', () => {
+    const repositoriesPage = new ChartRepositoriesPagePo(undefined, 'manager');
+
+    ChartRepositoriesPagePo.navTo();
+    repositoriesPage.list().resourceTable().sortableTable().checkVisible();
+    repositoriesPage.list().resourceTable().sortableTable().checkLoadingIndicatorNotVisible();
+    repositoriesPage.waitForPage();
+
+    // Ignoring the user profile picture
+    cy.hideElementBySelector('[data-testid="nav_header_showUserMenu"]');
+    // Ignore dinamic data inside rows 5 y 4(age and branch)
+    cy.hideElementBySelector("[data-testid^='sortable-cell-'][data-testid$='-5']");
```

Comment:
What are these rows exactly? The comment indicates a row but the id a cell, which one is actually true?

Reply:
This is hiding all the dynamic texts for column 5 and then for column 4


this is the result after executing the 2 methods calls.
<img width="989" alt="Screenshot 2025-05-16 at 3 53 59 p m" src="https://github.com/user-attachments/assets/35a992e9-7e66-4492-937e-f668cf515e4d" />
g the